addons/hr_expense/models/hr_expense.py

```diff
@@ -1353,6 +1353,14 @@ def action_register_payment(self):
 
     def action_open_expense_view(self):
         self.ensure_one()
+        if self.expense_number == 1:
+            return {
+                'type': 'ir.actions.act_window',
+                'view_mode': 'form',
+                'views': [[False, "form"]],
```

Comment:
You don't need to provide both `view_mode` and `views`, one (any) is enough if you take default views anyways

Reply:
done !